Signal/src/call/PeerConnectionClient.swift

```diff
@@ -115,7 +115,14 @@ class PeerConnectionClient: NSObject, RTCPeerConnectionDelegate, RTCDataChannelD
 
     private var videoSender: RTCRtpSender?
     private var localVideoTrack: RTCVideoTrack?
-    private weak var remoteVideoTrack: RTCVideoTrack?
+    // RTCVideoTrack is fragile and prone to throwing exceptions and/or
```

Comment:
I think audioTrack, videoSender, audioSender, suffer similarly when PeerConnectionClient holds a strong reference to them at the time RTCPeerConnection deallocs.